mysql-test/suite/perfschema/r/show_sanity.result

```diff
@@ -412,6 +412,7 @@ SHOW_MODE	SOURCE	VARIABLE_NAME
 5.6	I_S.SESSION_VARIABLES	INNODB_COMPRESSED_COLUMNS_THRESHOLD
 5.6	I_S.SESSION_VARIABLES	INNODB_COMPRESSED_COLUMNS_ZIP_LEVEL
 5.6	I_S.SESSION_VARIABLES	INNODB_DEADLOCK_DETECT
+5.6	I_S.SESSION_VARIABLES	INNODB_ENCRYPT_ONLINE_ALTER_LOGS
```

Comment:
Put `INNODB_ENCRYPT_ONLINE_ALTER_LOGS` as an exception to `show_sdanity.test`

Reply:
done